lib/expansion_tile_card.dart

```diff
@@ -95,64 +102,77 @@ class ExpansionTileCard extends StatefulWidget {
   /// Defaults to 2.0.
   final double elevation;
 
+  ///The elevetion when collapsed
+  ///
+  ///Defaults to 0.0
+  final double initialElevation;
+
+  final Color shadowColor;
+
   /// Specifies if the list tile is initially expanded (true) or collapsed (false, the default).
   final bool initiallyExpanded;
 
   /// The padding around the outside of the ExpansionTileCard while collapsed.
-  /// 
+  ///
   /// Defaults to EdgeInsets.zero.
   final EdgeInsetsGeometry initialPadding;
 
   /// The padding around the outside of the ExpansionTileCard while collapsed.
-  /// 
+  ///
   /// Defaults to 6.0 vertical padding.
   final EdgeInsetsGeometry finalPadding;
 
   /// The inner `contentPadding` of the ListTile widget.
-  /// 
+  ///
   /// If null, ListTile defaults to 16.0 horizontal padding.
   final EdgeInsetsGeometry contentPadding;
 
   /// The background color of the unexpanded tile.
-  /// 
+  ///
   /// If null, defaults to Theme.of(context).canvasColor.
   final Color baseColor;
 
   /// The background color of the expanded card.
-  /// 
+  ///
   /// If null, defaults to Theme.of(context).cardColor.
   final Color expandedColor;
 
   /// The duration of the expand and collapse animations.
-  /// 
+  ///
   /// Defaults to 200 milliseconds.
   final Duration duration;
 
   /// The animation curve used to control the elevation of the expanded card.
-  /// 
+  ///
   /// Defaults to Curves.easeOut.
   final Curve elevationCurve;
 
   /// The animation curve used to control the height of the expanding/collapsing card.
-  /// 
+  ///
   /// Defaults to Curves.easeIn.
   final Curve heightFactorCurve;
 
   /// The animation curve used to control the rotation of the `trailing` widget.
-  /// 
+  ///
   /// Defaults to Curves.easeIn.
   final Curve turnsCurve;
 
   /// The animation curve used to control the header, icon, and material colors.
-  /// 
+  ///
   /// Defaults to Curves.easeIn.
   final Curve colorCurve;
 
   /// The animation curve used by the expanding/collapsing padding.
-  /// 
+  ///
   /// Defaults to Curves.easeIn.
   final Curve paddingCurve;
 
+  ///This color will overwrite all other colors
+  final Color color;
+
+  ///This elevation will overwrite all other
+  final double constantElevation;
+
```

Comment:
Same here. I'm not sure why we need an override for elevation if you can just set `initialElevation` and `elevation` to the same value.

Reply:
Same as the color

Reply:
That's...really interesting. I'll have to give it a try to see why that would be.